executor/index_lookup_hash_join.go

```diff
@@ -293,16 +293,20 @@ func (e *IndexNestedLoopHashJoin) Close() error {
 		}
 		e.taskCh = nil
 	}
+	if e.runtimeStats != nil {
+		rootStats := e.ctx.GetSessionVars().StmtCtx.RuntimeStatsColl.GetRootStats(e.baseExecutor.id.String())
+		concurrency := len(e.joinChkResourceCh)
+		if concurrency > 1 {
+			rootStats.SetConcurrencyInfo(fmt.Sprintf("Concurrency:%d", concurrency))
+		} else {
+			rootStats.SetConcurrencyInfo(fmt.Sprintf("Concurrency:OFF"))
```

Comment:
Even if concurrency is 1, index lookup hash join still invoke a groutine. I think we should not show `OFF` here.